sdks/csharp/adtIotHub_delete.cs

```diff
@@ -19,14 +20,24 @@ namespace Samples.AdtIothub
     public static class DeleteDeviceInTwinFunc
     {
         private static string adtAppId = "https://digitaltwins.azure.net";
-        private static readonly string adtInstanceUrl = System.Environment.GetEnvironmentVariable("ADT_SERVICE_URL", EnvironmentVariableTarget.Process);
-        private static readonly HttpClient httpClient = new HttpClient();
+        private static readonly string adtInstanceUrl = Environment.GetEnvironmentVariable("ADT_SERVICE_URL", EnvironmentVariableTarget.Process);
+        private static readonly HttpClient singletonHttpClientInstance = new HttpClient();
 
         [FunctionName("DeleteDeviceInTwinFunc")]
         public static async Task Run(
             [EventHubTrigger("lifecycleevents", Connection = "EVENTHUB_CONNECTIONSTRING")] EventData[] events, ILogger log)
         {
-            var exceptions = new List<Exception>();
+            var exceptions = new List<Exception>(events.Length);
+
+            // Create Digital Twin client
+            var cred = new ManagedIdentityCredential(adtAppId);
+            var client = new DigitalTwinsClient(
```

Comment:
Silly to create the client in the loop when nothing changes for the client parameters inside the loop.